edge-apps/iframe/README.md

```diff
@@ -0,0 +1,79 @@
+# Creating an Edge App with iFrame in Screenly
+
+This guide will walk you through the steps to create an Edge App in Screenly that uses an iFrame to load another HTML page or dashboard.
+
+## Prerequisites
+
+* Install Screenly CLI - Please follow the guide [here](https://github.com/Screenly/cli)
+
+## Steps
+
+1. **Log in to the Screenly account via CLI**
+
+   Follow the on-screen instructions to log in to your Screenly account.
+
+   ```bash
+   screenly login
+   ```
+
+2. **Download and Open Edge App Playground**
+
+   ```bash
+   git clone https://github.com/Screenly/Playground.git
+   cd edge-apps/iframe
+   ```
+
+3. **Enter iFrame Edge App Directory**
+
+   ```bash
+   cd edge-apps/iframe
+   ```
+
+4. **Create a New iFrame Edge App:**
+
+   ```bash
+   screenly edge-app create --name iframePage --in-place
+   ```
+
+   > Replace "iframePage" with your desired app name.
+
+5. **Upload the Edge App**
+
+   ```bash
+   screenly edge-app upload
+   ```
+
+   > Wait for the upload to complete.
+
+6. **Specify the iFrame URL**
+
+   Replace "WEBPAGE_URL" with the actual URL you want to display in the iFrame.
+
+   ```bash
+   screenly edge-app setting set iframe=WEBPAGE_URL
+   ```
+
+   or
+
+   ```bash
+   screenly edge-app setting set iframe='<iframe src="WEBPAGE_URL" title="programiz pro website" height="500" width="500" ></iframe>'
```

Comment:
Why don't we just use the iFrameURL setting here rather than passing the entire tag because it is anyways being replaced by the other tag. That way we can avoid extracting the URL.

Reply:
It was suggested by @vpetersson to support both the iframe full tag and URL.

Reply:
Correct. We want to support both seamlessly.

Reply:
Ok, but if we are supporting the full tag, we might want to copy all the provided (supported) attributes maybe. Otherwise it does not add any value over just providing the URL.

Reply:
We don't want to pass on the hight/width attributes. Not sure if there are other attributes that we should retain. @salmanfarisvp can you check the valid attributes and see if there are some we should retain, as this is a valid point.

Reply:
Since we only support one asset at a time and the page content needs to be responsive depending on the screen, we should trim the iframe values and only render the URL.

Reply:
@vpetersson The main attributes are - height, width, allowfullscreen, and name. Since we can't accompany multiple screens at a time, I think we can put the iframe as full screen. what do you guys think?

Reply:
Yeah, let's just ignore those and pass on the actual URL.

Reply:
Noted.